src/api/app/models/path_element.rb

```diff
@@ -1,7 +1,9 @@
 class PathElement < ApplicationRecord
+  # FIXME: This should be called parent
```

Comment:
how are this FIXME related with the commit? :confused:

Reply:
It's mostly to help us in the future (to hopefully refactor this).

Reply:
That's fine, bit it is not related with the commit. IMO it should be in a different one, let's say it is a nitpick....

Reply:
It doesn't look like changing this would require a migration. So I would prefer to change the names instead of adding a fixme that will stay there forever.
And if there isn't time for this, let's open an issue. Maybe someone else wants to jump in.